lib/tic_tac_toe.ex

```diff
@@ -0,0 +1,49 @@
+defmodule TicTacToe do
+
+  def run() do
+    start(new_game())
+  end
+
+  defp new_game() do
+    UI.introduce_ttt()
+    %Game{
+      board: %Board{size: 3, cells: Board.create_cells(3), marks: ["X", "O"]},
+      current_player: UI.prompt_for_starter()
+    }
+  end
+
+  defp start(game) do
+    UI.print_board(game.board)
+    updated_board = Board.place_mark(game.board, HumanPlayer.choose_position(game.current_player), game.current_player)
+    if ongoing?(updated_board) do
+      start(%{game |
+        board: updated_board,
+        current_player: switch_player(game.current_player, game.board.marks)})
+    else
+      verdict(updated_board)
+      play_again()
+    end
+  end
+
+  defp verdict(board) do
```

Comment:
That sounds like a court of law 😄 "The jury is back, the verdict is... ... ... ... X wins the game!"